modules/ossm-deploying-multiple-control-planes.adoc

```diff
@@ -0,0 +1,152 @@
+
+This procedure demonstrates how to configure a cluster to host two control planes in separate system namespaces. Each control plane will manage specific application workloads running in designated namespaces. The namespaces are associated with a control plane using revision and discovery selector configurations.
+
+.Prerequisites
+* The {SMProduct} operator is installed.
+* An {istio} CNI resource is created.
+* The `istioctl` binary is installed on your localhost.
+
+.Procedure
+
+. Create the namespace for the first {istio} control plane called `istio-system-1`:
++
+[source,terminal]
+----
+$ oc create namespace istio-system-1
+----
+
+. Add label to the namespace:
++
+[source,terminal]
+----
+$ oc label namespace istio-system-1 usergroup=usergroup-1
+----
+
+. Create a YAML file named `istio-1.yaml` with the name `mesh-1` and the discoverySelector `usergroup-1`:
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: mesh-1
+spec:
+  namespace: istio-system-1
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            usergroup: usergroup-1
+----
+
+. Create `Istio` resource:
++
+[source,terminal]
+----
+$ oc apply -f istio-1.yaml
+----
+
+. Create a namespace for the second Istio control plane called `istio-system-2`:
++
+[source,terminal]
+----
+$ oc create namespace istio-system-2
+----
+
+. Add label to the second namespace:
++
+[source,terminal]
+----
+$ oc label namespace istio-system-2 usergroup=usergroup-2
+----
+
+. Create a YAML file named `istio-2.yaml`:
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: mesh-2
+spec:
+  namespace: istio-system-2
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            usergroup: usergroup-2
+----
+
+. . Create `Istio` resource:
++
+[source,terminal]
+----
+$ oc apply -f istio-2.yaml
+----
+
+. Deploy a policy for workloads in the `istio-system-1` namespace to only accept mutual TLS traffic `peer-auth-1.yaml`:
++
+[source,terminal]
+----
+$ oc apply -f peer-auth-1.yaml
+----
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: security.istio.io/v1
+kind: PeerAuthentication
+metadata:
+  name: "usergroup-1-peerauth"
+  namespace: "istio-system-1"
+spec:
+  mtls:
+    mode: STRICT
+----
+
+. Deploy a policy for workloads in the `istio-system-2` namespace to only accept mutual TLS traffic `peer-auth-2.yaml`:
++
+[source,terminal]
+----
+$ oc apply -f peer-auth-2.yaml
+----
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: security.istio.io/v1
+kind: PeerAuthentication
+metadata:
+  name: "usergroup-2-peerauth"
+  namespace: "istio-system-2"
+spec:
+  mtls:
+    mode: STRICT
+----
+
+.Verfication
+
+. Verify that both the control planes deploy and run:
```

Comment:
```suggestion
. Verify that both of the Istio control planes deploy and run. This can be done by validating that the `Istiod` pod is successfully running in each Istio system namespace.
```